src/assets/data/faqs.json

```diff
@@ -378,14 +378,14 @@
 				"questionId": "cna_resources",
 				"questionText": "What resources are available for CNAs",
 				"questionResponseParagraphs": [
-					"See <a href='/Support/Resources#CNA'>CNA Resources</a> for a complete list of documents and videos for CNAs."
+					"See <a href='/ResourcesSupport/Resources#CVENumberingAuthorities'>CNA Resources</a> for a complete list of documents and videos for CNAs."
 				]
 			},
 			{
 				"questionId": "github_submit",
 				"questionText": "How can my organization submit content via GitHub",
 				"questionResponseParagraphs": [
-					"See <a href='/Support/Resources#GitHubSubmissions'>GitHub Submissions</a> on the Resources page."
+					"See <a href='/ResourcesSupport/Resources#CVENumberingAuthorities'>GitHub Submissions</a> on the Resources page."
```

Comment:
Should this be `<a href='/ResourcesSupport/Resources#GitHubSubmissions'>` ? (Although it seems that page anchor doesnt actually jump to the Github section on the Resources page)

Reply:
The ticket said to change it in the way that I did because there is no anchor for GitHubSubmissions